test/core/rules/std_test.py

```diff
@@ -47,655 +59,26 @@ def assert_rule_pass_in_sql(code, sql, configs=None):
         )
 
 
-# ############## STD RULES TESTS
-@pytest.mark.parametrize(
-    "rule,pass_fail,qry,fixed,configs",
-    [
-        ("L001", "fail", "SELECT 1     \n", "SELECT 1\n", None),
-        ("L002", "fail", "    \t    \t    SELECT 1", None, None),
-        ("L003", "fail", "     SELECT 1", "SELECT 1", None),
-        ("L004", "pass", "   \nSELECT 1", None, None),
-        ("L004", "pass", "\t\tSELECT 1\n", None, None),
-        ("L004", "fail", "   \n  \t \n  SELECT 1", None, None),
-        ("L005", "fail", "SELECT 1 ,4", "SELECT 1,4", None),
-        ("L008", "pass", "SELECT 1, 4", None, None),
-        ("L008", "fail", "SELECT 1,   4", "SELECT 1, 4", None),
-        ("L008", "fail", "SELECT 1,4", "SELECT 1, 4", None),
-        ("L013", "pass", "SELECT *, foo from blah", None, None),
-        ("L013", "fail", "SELECT upper(foo), bar from blah", None, None),
-        ("L013", "pass", "SELECT *, foo from blah", None, None),
-        # Don't expect alias if allow_scalar = True (default)
```

Comment:
I think it's a shame we lost those comments in the process :( would it be too hard to add ? (in case you have it scripted)

Reply:
I'm adding them all manually... 😢

Reply:
Definitely need to keep them though I agree